README.md

```diff
@@ -158,8 +156,8 @@ We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX elem
 
 Kedro-Viz uses features flags to roll out some experimental features. The following flags are currently in use:
 
-| Flag | Description |
-|------| ------------|
+| Flag        | Description                                                                             |
```

Comment:
Did your formatting tool do this automatically? 😮  Please share your magical method.

Reply:
oh, but it's vs code magic :)

Reply:
😮 🪄 🔮